src/js/components/generateDetachedFiles/GenerateDetachedFilesPage.jsx

```diff
@@ -158,35 +158,20 @@ export default class GenerateDetachedFilesPage extends React.Component {
         this.setState({[file]:dFile});
     }
 
-    showError(file, header, description) {
-        // show error that occurs at any point during file generation
-		const state = Object.assign({}, this.state[file], {
-			error: {
-				show: true,
-				header: header,
-				description: description
-			}
-		});
-		
-		this.setState({
-			[file]: state
-		});
-	}
-
-    hideError(file) {
-        // stop displaying the error for the given file
-		const state = Object.assign({}, this.state[file], {
-			error: {
-				show: false,
-				header: '',
-				description: ''
-			}
-		});
-
-		this.setState({
-			[file]: state
-		});
-	}
+    updateError(file, header='', description='') {
```

Comment:
Yes we do, this also replaces hideError which will do a call with no parameters.